server/src/main/java/io/crate/expression/symbol/SymbolVisitors.java

```diff
@@ -193,6 +193,26 @@ protected Void visitSymbol(Symbol symbol, Void context) {
             }
             return null;
         }
+
+        @Override
+        public Void visitVoidReference(VoidReference symbol, Void context) {
```

Comment:
Same here - is it possible to re-use `visitReference` or depend on the base class default traversal implementation?

Reply:
I fixed the initial logic so when I try to re-use `visitReference` it requires a bit more effort, please have a look at the new commit.